primeqa/pipelines/README.md

```diff
@@ -16,146 +16,46 @@
-context = [["""Chemtrail conspiracy theory The chemtrail conspiracy theory is based 
-            on the erroneous belief that long-lasting condensation trails are 
-            \"chemtrails\" consisting of chemical or biological agents left in the 
-            sky by high-flying aircraft, sprayed for nefarious purposes undisclosed 
-            to the general public. Believers in this conspiracy theory say that while 
-            normal contrails dissipate relatively quickly, contrails that linger must 
-            contain additional substances. Those who subscribe to the theory speculate 
-            that the purpose of the chemical release may be solar radiation management,
-            weather modification, psychological manipulation, human population control, 
-            or biological or chemical warfare, and that the trails are causing 
-            respiratory illnesses""",
-            """Associated with jet streams is a phenomenon known as clear-air turbulence 
-            (CAT), caused by vertical and horizontal wind shear caused by jet streams. 
-            The CAT is strongest on the cold air side of the jet, next to and just under 
-            the axis of the jet. Clear-air turbulence can cause aircraft to plunge and so 
-            present a passenger safety hazard that has caused fatal accidents, such as the 
-            death of one passenger on United Airlines Flight 826. 
-            Section: Uses.:Possible future power generation.""",
-            """Contrails are a manmade type of cirrus cloud formed when water vapor from 
-            the exhaust of a jet engine condenses on particles, which come from either the 
-            surrounding air or the exhaust itself, and freezes, leaving behind a visible trail. 
-            The exhaust can also trigger the formation of cirrus by providing ice nuclei 
-            when there is an insufficient naturally-occurring supply in the atmosphere. 
-            One of the environmental impacts of aviation is that persistent contrails can 
-            form into large mats of cirrus, and increased air traffic has been implicated 
-            as one possible cause of the increasing frequency and amount of cirrus"""]]
-answers = fid_reader.predict(question,context)  
-print(json.dumps(answers, indent=4)) 
+queries=["Which country is Canberra located in?"]
+answers = pipeline.run(query)
 ```
 
-## QA Pipeline
+## Long Form Question Answering (LFQA) 
 
-The QA pipeline is used for Open Retrieval Question Answering. Open retrieval systems query large document stores for relevant passages. Long Form Question Answering (LFQA) is a generative task where the retrieved passages are used to generate a complex multi-sentence answer.
+LFQA is a generative task where the retrieved passages are used to generate a complex multi-sentence answer.
 
 In this example we show a QA Pipeline using a ColBERT retriever and a Fusion in Decoder (FID) generator.
 
```

Comment:
In this ReadMe we should also give an example of a  pipeline supported via LLM Readers and/or point to notebooks here https://github.com/primeqa/primeqa/blob/prompt_reader/notebooks/retriever-reader-pipelines/